### ISSUE TITLE
updateInfos failed Error: Error: spawn git ENOENT

### ISSUE THREAD

**Issue by keilladraconis** [rvion/CushyStudio#134]
## Describe the bug

OS: Windows 10

When I start CushyStudio, it all seems to work, however the tab next to the couch at the top left is forever spinning and contains an error message:
```
updateInfos failed Error: Error: spawn git ENOENT at __node_internal_captureLargerStackTrace (node:internal/errors:496:5) at __node_internal_errnoException (node:internal/errors:623:12) at ChildProcess._handle.onexit (node:internal/child_process:284:19) at onErrorNT (node:internal/child_process:477:16) at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
```

It seems like it's trying to launch Git, but cannot. 

I do have git installed and it show up on both PowerShell and Command shell.

**Comment by rvion** [comment]
thanks for the feedback ! that's indeed problematic. 
so, Cushy fails to retrieve available updates. is the update script at the root working ?

I'll try to priorize removing the need for git asap

**Comment by keilladraconis** [comment]
Yes, the update script at the root does work. I hate windows, I could figure this issue out on my own on a linux system but windows' pathing is a mystery to me.

**Comment by keilladraconis** [comment]
Alright, well after a bit of thinking I dug around in the batch files and tried the following:
```
SET GIT_INSTALL_DIR=C:\Program Files\Git\cmd
SET CODE_INSTALL_DIR=C:\Users\keilla\AppData\Local\Programs\Microsoft VS Code\bin
SET "PATH=%NODE_INSTALL_DIR%;%GIT_INSTALL_DIR%;%CODE_INSTALL_DIR%;%PATH%"
```
There's some kind of confusing re-setting of the path in this script, also alternation between different methods of variable substitution. I think sticking to the fully-quoted `%var%` is best.

I'm unsure why setting paths this way matters. As you can see vsCode is installed as a user program, where git is installed as a everyone-program.

**Comment by rvion** [comment]
> There's some kind of confusing re-setting of the path in this script, also alternation between different methods of variable substitution. I think sticking to the fully-quoted %var% is best.

I'm clearly not good with windows scripting, since I don't use windows myself neither; I'll be more than happy to merge any PR from someone who knows how to make that more robust x)
